include/amici/steadystateproblem.h

```diff
@@ -54,26 +53,142 @@ class SteadystateProblem {
     void workSteadyStateBackwardProblem(Solver *solver, Model *model,
                                         const BackwardProblem *bwd);
 
+    /**
+     * @brief Returns the stored SimulationState
+     * @return stored SimulationState
+     */
+    const SimulationState &getFinalSimulationState() const {
+        return state_;
+    };
+
+    /**
+    * @brief Returns the quadratures from pre- or postequilibration
+    * @return xQB Vector with quadratures
+    */
+    const AmiVector &getEquilibrationQuadratures() const {
+        return xQB_;
+    }
+    /**
+     * @brief Returns state at steadystate
+     * @return x
+     */
+    const AmiVector &getState() const {
+        return x_;
+    };
```

Comment:
It has been there before, but do we need `x_` in addition to `state_.x`? (Same for `sx`)

Reply:
Probably not!